src/main/resources/static/PureCssLoginPage.css

```diff
@@ -0,0 +1,39 @@
+body {
```

Comment:
Read the comments on the PR. Generally we want to name this something shorter and with a friendlier, non-uppercased name. Consider say, ```login.css```